RecoMTD/TimingIDTools/src/TrackPUIDMVA.cc

```diff
@@ -0,0 +1,63 @@
+#include "RecoMTD/TimingIDTools/interface/TrackPUIDMVA.h"
+
+TrackPUIDMVA::TrackPUIDMVA(std::string weights_file) {
+  vars_.push_back("pt");
+  vars_.push_back("eta");
+  vars_.push_back("phi");
+  vars_.push_back("chi2");
+  vars_.push_back("ndof");
+  vars_.push_back("numberOfValidHits");
+  vars_.push_back("numberOfValidPixelBarrelHits");
+  vars_.push_back("numberOfValidPixelEndcapHits");
+  vars_.push_back("btlMatchChi2");
+  vars_.push_back("btlMatchTimeChi2");
+  vars_.push_back("etlMatchChi2");
+  vars_.push_back("etlMatchTimeChi2");
+  vars_.push_back("mtdt");
+  vars_.push_back("path_len");
+
+  std::string options("!Color:Silent");
+  std::string method("BDT");
+
+  mva_ = std::make_unique<TMVAEvaluator>();
+  mva_->initialize(options, method, weights_file, vars_, spec_vars_, true, false);  //use GBR, GradBoost
```

Comment:
Yes it works with GBR so we use this as default, but it may be good to have the functionality to use TMVA if eg different training is used in the future